scripts/about.js

```diff
@@ -1,8 +1,4 @@
-#!/usr/bin/env node
```

Comment:
don't mind, but curious as to why you removed the shebang.  Was something not working?

Reply:
Only reason I removed it was because in the gulp cleanup PR I added an npm script which runs `node scripts/about.js` and so we don't really need to be able to execute this file directly. Open to adding it back tho if desired